### ISSUE TITLE
Code references non-existing images from the bundle

### ISSUE THREAD

**Issue by zoul** [kolinkrewinkel/KKGridView#108]
The `KKGridViewCell` references images that are not present in the bundle (such as `UIPreferencesWhiteCheck`). Is this a known issue?

**Comment by kolinkrewinkel** [comment]
Yep; I think I'll end up base64 encoding the images we want to bundle with the project.
